templates/admin/src/tab-app.tsx_jsx.ts

```diff
@@ -0,0 +1,77 @@
+import { TemplateFunction } from "../../../src/lib/createAdapter";
```

Comment:
Would it make sense to integrate this into component into the tab.tsx.ts?

Reply:
We could do this, I simply kept the same architecture as we have for index.tsx and app.tsx. Which is also the architecture suggested by adapter-react.

Another question is, whether we should switch to functional components. I use them now in ham and it seems easier to write code.